cmake/modules/contrib/BLAS.cmake

```diff
@@ -55,3 +55,11 @@ elseif(USE_BLAS STREQUAL "none")
 else()
   message(FATAL_ERROR "Invalid option: USE_BLAS=" ${USE_BLAS})
 endif()
+
+if(USE_MKL_DNN STREQUAL "ON")
+  find_library(BLAS_LIBRARY_MKLDNN dnnl)
+  list(APPEND TVM_RUNTIME_LINKER_LIBS ${BLAS_LIBRARY_MKLDNN})
+  list(APPEND RUNTIME_SRCS ${CBLAS_CONTRIB_SRC})
+  add_definitions(-DUSE_MKL_DNN=1)
+  message(STATUS "Use MKL DNN library " ${BLAS_LIBRARY_MKLDNN})
+endif()
```

Comment:
What's the strategy if both `USE_BLAS` AND `USE_MKL_DNN` are set?

Reply:
Update the logic here. MKL DNN will only be used when USE_BLAS is not `none`. When both are set, MKL DNN will be only used in the sgemm op as the library has limited support for the BLAS operators. And I find that MKL DNN kernel achieves better performance than MKL in TVM.

Reply:
Another minor nit is to change `USE_MKL_DNN` to either `USE_DNNL` or `USE_MKLDNN`. The first one aligns with the renaming trend of the library. The second one follows the coding convention in MKL-DNN before renaming and in other projects like MXNet.

Reply:
fixed

Reply:
Thanks for the quick turnaround.